### ISSUE TITLE
SonataMediaBundle incompatible with Twig 2

### ISSUE THREAD

**Issue by enekochan** [sonata-project/SonataMediaBundle#1183]
#### Sonata packages

```bash
$ composer show sonata-project/*
sonata-project/admin-bundle              3.12.0          The missing Symfon...
sonata-project/block-bundle              3.3.0           Symfony SonataBloc...
sonata-project/cache                     1.0.7           Cache library
sonata-project/classification-bundle     3.x-dev af4332c Symfony SonataClas...
sonata-project/core-bundle               3.2.0           Symfony SonataCore...
sonata-project/datagrid-bundle           2.2             Symfony SonataData...
sonata-project/doctrine-extensions       1.0.2           Doctrine2 behavior...
sonata-project/doctrine-orm-admin-bundle 3.x-dev 3e3cbdb Symfony Sonata / I...
sonata-project/easy-extends-bundle       2.1.10          Symfony SonataEasy...
sonata-project/exporter                  1.7.0           Lightweight Export...
sonata-project/intl-bundle               2.3.0           Symfony SonataIntl...
sonata-project/media-bundle              3.x-dev cbf60b6 Symfony SonataMedi...
sonata-project/notification-bundle       3.0.0           Symfony SonataNoti...
```

#### Symfony packages

```bash
$ composer show symfony/*
symfony/assetic-bundle     v2.8.1  Integrates Assetic into Symfony2
symfony/monolog-bundle     v2.12.1 Symfony MonologBundle
symfony/polyfill-apcu      v1.3.0  Symfony polyfill backporting apcu_* func...
symfony/polyfill-intl-icu  v1.3.0  Symfony polyfill for intl's ICU-related ...
symfony/polyfill-mbstring  v1.3.0  Symfony polyfill for the Mbstring extension
symfony/polyfill-php54     v1.3.0  Symfony polyfill backporting some PHP 5....
symfony/polyfill-php55     v1.3.0  Symfony polyfill backporting some PHP 5....
symfony/polyfill-php56     v1.3.0  Symfony polyfill backporting some PHP 5....
symfony/polyfill-php70     v1.3.0  Symfony polyfill backporting some PHP 7....
symfony/polyfill-util      v1.3.0  Symfony utilities for portability of PHP...
symfony/security-acl       v3.0.0  Symfony Security Component - ACL (Access...
symfony/swiftmailer-bundle v2.4.2  Symfony SwiftmailerBundle
symfony/symfony            v2.8.16 The Symfony PHP framework
```

#### PHP version

```bash
$ php -v
PHP 7.0.15 (cli) (built: Jan 20 2017 06:03:19) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.15, Copyright (c) 1999-2017, by Zend Technologies
    with Xdebug v2.5.0, Copyright (c) 2002-2016, by Derick Rethans
```

## Subject

Incompatibility with Twig 2 as the Twig extensions defined in SonataMediaBundle still have the `getName()` method that has been deprecated.

## Steps to reproduce

Upgrade from Twig 1 to Twig 2 in a Symfony project for example.

## Expected results

No errors.

## Actual results

An error like this:

```
The "sonata_media" extension is not enabled.
```

**Comment by greg0ire** [comment]
Can you make a PR?

**Comment by enekochan** [comment]
I'll do as soon as possible. Should I target branch 3.x or master?

**Comment by greg0ire** [comment]
3.x (this will hopefully be backwards-compatible)

**Comment by OskarStark** [comment]
I created a PR

**Comment by enekochan** [comment]
Fixed in #1184
